spec/spec_helper.rb

```diff
@@ -50,8 +51,26 @@ def require_relative(path)
   end
 
   config.fixture_path = "#{::Rails.root}/spec/fixtures"
-  config.use_transactional_fixtures = true
-  config.use_instantiated_fixtures  = false
+
+  config.before(:suite) do
+    DatabaseCleaner.clean_with(:truncation)
+  end
+
+  config.before(:each) do
+    DatabaseCleaner.strategy = :transaction
```

Comment:
Does this before(:each) always run before the one of `:type => worker` in the event of a worker spec?

In the non-worker case, is this the same was what we did before?
```
config.use_transactional_fixtures = true
config.use_instantiated_fixtures  = false
```

Reply:
> In the non-worker case, is this the same was what we did before?

Trust me, I tried... for a long time, and dug into the internals of all of it.  It isn't well documented, even in the `DatabaseCleaner` README:

https://github.com/DatabaseCleaner/database_cleaner#rspec-with-capybara-example

But there is a code sample in there that just removes the whole config setting.

Basically, that config setting comes from Rails (it is just wrapped by `rspec-rails`), and there isn't a good way to disable it in one spot, and then re-enable it in another.  A lot of it is hacking around the fact that the Rails bit is meant for `TestUnit`/`Minitest`, and `rspec-rails` is building a wrapper abstraction layer around it that.

Reply:
Sorry, I miss read.  This is doing what we did before (hopefully).

I thought the question was "can't we just do what we did before".